### PR TITLE
Added kilojoules property to ingredient model (Issue #568)

### PR DESCRIPTION
**Pull request by nopinter**
### Proposed Changes

  - added "energy_kilojoule" property to Ingredient class in wger/nutrition/models.py

  - added kilojoules to display on ingredient details page in wger/nutrition/templates/ingredient/view.html

  - added to wger/nutrition/static/js/nutrition.js, making kilojoules responsive to updates in ingredient amount according to user input

  - added test to wger/nutrition/tests/test_ingredient.py, verifying correct kilojoule conversion

### Please check that the PR fulfills these requirements

- [✓] Tests for the changes have been added (for bug fixes / features)
- [✓] New python code has been linted with with flake8 (``flake8 --config .github/linters/.flake8``)
and isort (``isort``) 
- [✓] Added yourself to AUTHORS.rst

### Other questions

* Does this PR introduce a breaking change such as a database migration? (i.e.
what changes might users need to make in their running application due to
this PR?) 

No

### CODE REVIEW

**Comment by nopinter** [comment]
I worked on these changes with my partner, @derekli17, which is why his name is also included in AUTHORS.rst. We are currently undergraduates studying computer science, so if you have any feedback or want us to make changes, please feel free to let us know! Thanks.

File: wger/nutrition/models.py [hunk 1]
```diff
@@ -512,6 +512,17 @@ def get_owner_object(self):
         """
         return False
 
+    def _get_energy_kilojoule(self):
+        """
+        returns kilojoules for current ingredient, 0 if energy is uninitialized
+        """
+        if self.energy:
+            return self.energy * 4184
```
Comment (rolandgeider) [inline wger/nutrition/models.py:520]: Sorry, 1kcal ist 4.1 kJ, not four thousand! :D I had just copy pasted it from the German wikipedia and hadn't thought that not everybody uses commas for this

File: wger/nutrition/models.py [hunk 2]
```diff
@@ -512,6 +512,17 @@ def get_owner_object(self):
...
+        else:
+            return 0
+
+    energy_kilojoule = property(_get_energy_kilojoule)
```
Comment (rolandgeider) [inline wger/nutrition/models.py:524]: You can just define the property directly

```python
@property
def energy_kilojoule(self):
    if self.energy:
    ...
```

**Review by rolandgeider — CHANGES_REQUESTED** [review]

**Comment by rolandgeider** [comment]
While showingeverywhere both the kcal and the kJ would break the UI flow, I'd add them to the nutrition list and the nutritional data table under the plan, then this would be ready to merge

**Comment by nopinter** [comment]
Hi, thanks for your feedback! We've implemented your suggested changes. We found the "4,184" / "4.184" issue very funny, but that now is fixed 👍 . Additionally, we wanted your feedback on an aesthetic issue, which we've attached a picture of. 

Screen Shot 2020-12-13 at 10 30 05 PM

As you can see in the red circle, sometimes the kilojoules wrap around onto the next line. Let us know your thoughts on this. We'd be more than willing to change some of the HTML/CSS to make it look cleaner if necessary.

**Comment by rolandgeider** [comment]
I also had to laugh when I opened a plan and saw like half a million kJ 😂

And as for the line wrapping issue, we can save some space by removing the decimal place but other than that I wouldn't change much

**Review by rolandgeider — APPROVED** [review]

**Comment by nopinter** [comment]
We have experimented with removing the decimal place, but we are still observing the line wrapping issue. We didn't make any further changes for this reason

**Comment by nopinter** [comment]
Also, we would be willing to take #581 instead of #575 ! I'll will comment on those to reflect this

**Review by rolandgeider — APPROVED** [review]

**Comment by rolandgeider** [comment]
Merged, thanks!!

**Pull request merged**